docs/language-highlight.md

```diff
@@ -8,4 +8,25 @@
 <script src="//cdn.jsdelivr.net/npm/prismjs/components/prism-php.min.js"></script>
 ```
 
+To use the new languages, make sure the code block label matches the part after `prism-` in the file name. FOr example, for `prism-bash.js` write a code block labeled with `bash` like this:
+
+````
+```bash
+echo "hello"
+```
+````
+
+?> Note that with GitHub-flavored markdown, `sh` and `bash` are effectively aliases of each other, but this is not the case with Prism. So using `sh` will not enable `bash` syntax in this case.
+
+For `prism-php.js`, it would be:
```

Comment:
Do we need two examples? If we want two, why not use one of the built-in languages (`css`, `html`, or `js`) and one that requires loading an additional language?